vim/plugin-managers.md

```diff
@@ -0,0 +1,26 @@
+# Plugin Managers
+
+Today I learned that vim has two main plugin managers:
```

Comment:
There's also the classic https://packages.debian.org/sid/vim-addon-manager from 2007.